test/views/declaration/DeclarationAdditionalActorsViewSpec.scala

```diff
@@ -100,7 +100,7 @@ class DeclarationAdditionalActorsViewSpec extends UnitViewSpec with CommonMessag
 
       "display both 'Add' and 'Save and continue' button on page" in {
         val addButton = view.getElementsByAttributeValueMatching("name", Add.toString).first()
-        addButton.text() mustBe "site.addsupplementary.additionalActors.add.hint"
+        addButton.text() mustBe "site.adddeclaration.additionalActors.add.hint"
```

Comment:
I had a similar test to fix. I opted for two assertions -> text() must include (site.add) and text() must include(hint)